Opencast-Moodle/moodle-block_opencast
# Add default value to the oc import videos checkbox in Moodle core import wizard

This PR fixes #333,

### How it works:
- A new setting is added after (and depending on) "Allow video import within Moodle core course import wizard" called "Default value for opencast import videos' checkbox".
- The value set for this setting works as a default value for the checkbox in the Moodle core import wizard stating to "Include opencast video from X instance".
- Teachers who prefer to skip the steps, will now take advantage of this value, whether include or not to include opencast videos in the backup and restore.
- As to maintain the current behavior, this default value is set to `Unchecked` initially.

### To Test this PR:
- Make sure you have the latest plugins installed.
- Install this patch, and in the adding setting page of block_opencast, try to set value for the newly added setting mentioned above.
- While backing up a course, the value you have set as default must be applied in include opencast videos checkkbox for the instance in the import wizard sub-step. Perform the backup and restore without changing the oc checkbox in wizard and let the default value do its job.
- Try to change that default setting in the admin, and redo the backup and retore but this time use the Jump to final step and see the result.
- Feel free to test all the scenarios comes to your mind, but at the the default value must be applied and be effective in the way it has to be.
